File: Backend/router/StreamRouter.js
const express = require('express');
const streamRouter = express.Router();
const { streamVideo } = require('../Controllers/streamController');

/**
 * @route   GET /api/stream
 * @desc    Stream video from URL using FFmpeg
 * @access  Public
 */
streamRouter.get('/api/stream', streamVideo);

module.exports = streamRouter;

File: python-backend/routes/stream_router.py
from flask import Blueprint
from controllers.stream_controller import stream_video

# Create blueprint
stream_bp = Blueprint('stream', __name__)

@stream_bp.route('/api/stream', methods=['GET'])
def stream():
    """
    GET /api/stream?url=<stream_url>
    Stream video from URL using FFmpeg
    Query params:
        - url: Video stream URL (RTSP, HTTP, HTTPS, .mp4, .m3u8, etc.)
    """
    return stream_video()